docs/support-dump.md

```diff
@@ -0,0 +1,56 @@
+# Support Insight
+
+In many cases, especially when looking for [commercial support](commercial-support.md) there's a need for share the cluster state with other people.
+While one could always give access to the live cluster that is not always desired nor even possible.
+
+For those kind of cases we can lean on the work our friends at [troubleshoot.sh](https://troubleshoot.sh) have done.
+
+With troubleshoot tool you can essentially take a dump of the cluster state and share it with other people. You can even use [sbctl](https://github.com/replicatedhq/sbctl) tool to make the dump tarball to act as Kubernetes API.
+
+Let's look at how this works with k0s.
+
+## Setting up
+
+To get started you need to have `kubectl` and the admin kubeconfig. `support-bundle` itself is a [krew](https://krew.sigs.k8s.io/) plugin for kubectl. To set it up, follow the [instructions](https://troubleshoot.sh/docs/#installation).
```

Comment:
krew isn't necessary, you can download the binary release from https://github.com/replicatedhq/troubleshoot/releases and run the `support-bundle` binary separately. 

this might be preferable to setting up krew just to install `support-bundle` :smile_cat: 

we do plan to cut down on the number of binaries being released eventually but the one you're interested in is `support-bundle_linux_<arch>.tar.gz`

Reply:
Oh, good point. All the docs referred to krew thus I didn't even try by downloading it separately 🙄 

I'll rephrase this part